### ISSUE TITLE
 “Links do not have a discernible name” warning thrown by lighthouse

### ISSUE THREAD

**Issue by Rafi993** [thepracticaldev/dev.to#8668]
**Describe the bug**

Warning  “Links do not have a discernible name” is thrown by lighthouse. It is due to missing aria label in the featured article link

**To Reproduce**
 1. Go to dev.to
 2. Open up lighthouse in chrome dev tools
 3. Enable "accessibility" and run audit for "desktop"
 4. You will see the warning

**Expected behavior**
It should not throw the above warning

**Screenshots**

![warning](https://user-images.githubusercontent.com/8092120/84593103-cd73bc00-ae67-11ea-9057-ec510eea7fd8.png)

**Desktop:**
- OS: Ubuntu 18.04.4 LTS
- Browser: Brave
- Version: 1.9.80 Chromium: 81.0.4044.138 (Official Build) (64-bit)

**Comment by Rafi993** [comment]
Hi there if this issue is up for grabs I would like to help

**Comment by citizen428** [comment]
Thanks, @Rafi993, I assigned you the issue now. I labelled it as bug, because I think accessibility issues should always be resolved, especially when the fix is straightforward.

**Comment by Rafi993** [comment]
Awesome. Thanks @citizen428
